packages/components/FilePreview/FilesPreviewsContainer.tsx

```diff
@@ -27,6 +28,7 @@ export const FilesPreviewsContainer: React.FC<FilePreviewContainerProps> = ({
   onDelete,
   onDeleteGIF,
   onAudioUpdate,
+  showSmallPreview = false,
```

Comment:
I don't think this is ever set to anything other than false.
Please help me understand why this prop is needed (and where is being used)

Reply:
yeah it is mostly false, but in mini comment input it is true